### ISSUE TITLE
Histogram combines last 2 bins

### ISSUE THREAD

**Issue by jaybaker** [yhat/ggplot#404]
I setup a very simple dataframe to test a histogram edge condition I was seeing. The last two bins appear to be grouped together.

```
import pandas as pd
from ggplot import *
df = pd.DataFrame(dict(value=pd.Series([0,1,2,3,4,5]))
ggplot(aes(x='value'), data=df) + geom_histogram(binwidth=1)
```

This produces the following plot.
![ggplot histogram example](https://cloud.githubusercontent.com/assets/69569/5986312/00a5e2f2-a8b9-11e4-91e2-9b6ac87c0be7.png)

It should be 6 bins, each with a count of 1.

**Comment by lemahdi** [comment]
I have the same issue. Any progress?

**Comment by mgdadv** [comment]
This does not happen with R ggplot2. Here is the R code for the example:

```R
library(ggplot2)
dat=seq(0,5)
df=data.frame(value=dat)
ggplot(df, aes(x=value)) + geom_histogram(binwidth=1, color="white")
```

The R code pads the data values with an extra bin at the ends of the range. See 2f04679.

**Comment by glamp** [comment]
fixed in 0.9.3
